x-pack/plugins/infra/public/pages/metrics/hosts/components/host_details_flyout/flyout_wrapper.tsx

```diff
@@ -21,14 +21,14 @@ export interface Props {
 
 export const FlyoutWrapper = ({ node: { name }, closeFlyout }: Props) => {
   const { source } = useSourceContext();
-  const { searchCriteria } = useUnifiedSearchContext();
+  const { parsedDateRange } = useUnifiedSearchContext();
   const [hostFlyoutState, setHostFlyoutState] = useHostFlyoutUrlState();
 
   return source ? (
     <AssetDetails
       asset={{ id: name, name }}
       assetType="host"
-      dateRange={searchCriteria.dateRange}
+      dateRange={parsedDateRange}
```

Comment:
With this change, we make sure the flyout will always open with the same date range. Previously, if the date picker had relative dates, each time the flyout opened a new absolute date would be calculated.